commands/roll.go

```diff
@@ -0,0 +1,88 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/bwmarrin/discordgo"
```

Comment:
Move `"github.com/bwmarrin/discordgo"` below standard imports to be consistent with other files.